sql/2020/17_JAMstack/distribution_of_page_weight_requests_and_co2_grams_per_ssg_web_page.sql

```diff
@@ -0,0 +1,53 @@
+#standardSQL
+# Distribution of page weight, requests, and co2 grams per SSG web page
```

Comment:
This was a CMS chapter query and I'm not sure how relevant it is in the Jamstack chapter. Has the author asked for this data?

Reply:
Yeah :) I was thinking that it might be useful cc @ahmadawais.

Reply:
I believe it might be useful as that's the Jamstack value prop.